src/components/dashboard-container/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {
+  categoryCreate,
+
+} from '../../action/category-actions.js';
+import CategoryForm from '../category-form/'
+import { connect } from 'react-redux';
+import CategoryItem from '../../components/category-item'
+
+class DashboardContainer extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    console.log('mounted');
+  }
+
+  render() {
+    return (
+      <section>
+        <h2>Dashboard</h2>
+        <CategoryForm buttonText='Create' onComplete={this.props.categoryCreate} />
+        {this.props.categories.map((category) => {
+          return (
+            <CategoryItem categoryDelete={this.props.categoryDelete} categoryUpdate={this.props.categoryUpdate} category={category} />
```

Comment:
key needs to be added to the element or component which is going to be repeated, in this case, CategoryItem is been invoked multiple times not 'li'.